Automattic/wp-calypso
# Storage Add Ons: Create new Add Ons Package

Related to https://github.com/Automattic/wp-calypso/pull/79041#discussion_r1279224891
Tracking https://github.com/Automattic/martech/issues/2023

## Proposed Changes

* Initializes a new `add-ons` package to house any logic related to Calypso add-ons

## Testing Instructions



* Ensure that the Calypso bundle can still be generated properly ( check out this branch and run `yarn start` )

## Pre-merge Checklist
